web/lib/service/analytics.ts

```diff
@@ -33,10 +64,14 @@ export const withTracking =
 
 export async function identifyUser(userId: string) {
   setUserId(userId)
+  // @ts-expect-error Sprig doesn't yet have a native typescript snippet
+  window.Sprig.setUserId(userId)
```

Comment:
If you've moved sprig to its own file, you could export the Sprig variable and use it here.
E.g. at the bottom of `/sprig.ts`: `export const Sprig = window.Sprig as any`
Then, this line would be simply: `Sprig.setUserId(userId)`